receiver/netflowreceiver/config.go

```diff
@@ -0,0 +1,65 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package netflowreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/netflowreceiver"
+
+import "fmt"
+
+// Config represents the receiver config settings within the collector's config.yaml
+type Config struct {
+	// The scheme defines the type of flow data that the listener will receive
+	// The scheme must be one of sflow, netflow, or flow
+	Scheme string `mapstructure:"scheme"`
+
+	// The hostname or IP address that the listener will bind to
+	Hostname string `mapstructure:"hostname"`
+
+	// The port that the listener will bind to
+	Port int `mapstructure:"port"`
+
+	// The number of sockets that the listener will use
+	Sockets int `mapstructure:"sockets"`
```

Comment:
It took me a while to figure out what `sockets` was here but I guess on Linux you can use `SO_REUSEPORT` to allow multiple UDP sockets to be bound to the same underlying port. TIL